compiler/rustc_error_codes/src/error_codes/E0546.md

```diff
@@ -0,0 +1,11 @@
+A feature name is missing.
+
+Erroneous code example:
+
+```ignore (error only exists when building std)
+#[unstable(issue = "none")] // invalid
+#[stable(since = "a")] // invalid
+fn foo() { }
+```
+
+Provide a feature name.
```

Comment:
Just realized: might be nice to at least have a working example in here. :)

Reply:
I see and found using `[feature(staged_api)]` attribute to resolve previous compile error.
Thanks!